get_weather_data.ipynb

```diff
@@ -0,0 +1,98 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "963fbd82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import meteostat as ms\n",
+    "from datetime import datetime\n",
+    "from string import punctuation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bd1c821b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_weather_data(place,start,end=datetime.now):\n",
+    "    \n",
+    "    '''\n",
+    "    In the future should implement more in the way of processing for place to make sure it's in a form that will be useable.\n",
```

Comment:
I would recommend a more thorough docstring here to explain the sorts of inputs that the function can handle (especially the `place` string, which seems very open ended)

Here are some docstring examples: https://www.geeksforgeeks.org/python-docstrings/

My favorite is the Numpydoc standard